src/assetstest.json

```diff
@@ -6,16 +6,16 @@
         "cycle": "10",
         "year": "21",
         "collateral": "WETH",
-        "contract": {
-          "address": "0x4707b810b75C2DF8Ffeb50f7f33Af617Fbf3d766"
+        "lsp": {
+          "address": "0x4707b810b75C2DF8Ffeb50f7f33Af617Fbf3d766",
+          "library": "0x"
         },
-        "pool": {
+        "pools":[{
           "address": "",
           "location": "sushiswap"
```

Comment:
Do we need types for these pools?

Reply:
What do you mean by type? Protocol (uniswap, balancer, etc.) or are you saying to further drill down further into a type ( ie. Balancer Weighted Pool)

Reply:
I thought about the token type inside the pool (e.g. long / short token).

Reply:
I think ideally we have the minimum amount of information in the assets.json, in this case which are the incentivized pools and then query the contract for the rest of the info. That is not how it is now, but I think that should be the goal that way we don't have users making assumptions that could be outdated or incorrect on chain.

I could be convinced that we add that into the interface though since the above has not been a design goal up to this point.

Reply:
Hm.. since we will only go with one pool for now it is okay.
It might however be helpful if we have multiple pools but until then we already might have everything on-chain.

Reply:
> Hm.. since we will only go with one pool for now it is okay. It might however be helpful if we have multiple pools but until then we already might have everything on-chain.

This just prompted me to think about how we plan on incentivizing pools by different amounts which wouldn't be , that will need to be included here. So I think you are onto something here with a pool type or at least an incentive weight.

Reply:
Let's consider resolved for the sake of this PR, but you are right that this will need to be updated soon. I've opened #89 to track this issue. Resolve this conversation if you are good with that.